serverless/messaging/reference-content/snssqs-overview.mdx

```diff
@@ -13,9 +13,9 @@ dates:
   posted: 2023-01-04
 ---
 
-## What is SNS/SQS?
+## What is SQS/SNS?
 
-SNS and SQS are both managed messaging services developed by AWS and supported by Scaleway Messaging and Queuing. Although they are two separate services, they are often discussed simultaneously since they are frequently coupled to work together. A Scaleway Messaging and Queuing SNS/SQS namespace supports both services simultaneously.
+SQS and SNS are both managed messaging services developed by AWS and supported by Scaleway Messaging and Queuing. Although they are two separate services, they are often discussed simultaneously since they are frequently coupled to work together. A Scaleway Messaging and Queuing SNS/SQS namespace supports both services simultaneously.
 
 ### SNS
```

Comment:
If we change the order, should not we do it completely and set SQS first in 22 ...
What do you think ?

Reply:
Done